src/node/url.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Where does the unsafe assignment error come from? I would suggest moving the disable to that line.

Reply:
There are 2 lines, specifically `const XX = globalThis.XX`